packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/actions/ActionsTests.java

```diff
@@ -148,4 +151,71 @@ public void testGetActionDefinitionById() throws Exception
         restActionDefinition.getDescription().equals("This will add an aspect to the matched item.");
         restActionDefinition.getTitle().equals("Add aspect");
     }
+
+    @TestRail(section = {TestGroup.REST_API, TestGroup.ACTIONS}, executionType = ExecutionType.SANITY,
+            description = "Sanity test for ACTIONS endpoint GET action-conditions/{actionConstraintName}")
+    @Test(groups = {TestGroup.REST_API, TestGroup.ACTIONS, TestGroup.SANITY})
+    public void testGetSingleActionConstraint()
+    {
+        final UserModel testUser = dataUser.createRandomTestUser();
+        restClient.authenticateUser(testUser);
+
+        final String compareOperationsName = "ac-compare-operations";
+        final RestActionConstraintModel actionConstraintCompareOperations = restClient.
+                withCoreAPI().
+                usingActions().
```

Comment:
It looks weird to me to have the dots at the end of the lines - not at the start. I'm not sure we have any convention about it though.

Reply:
Thanks for catching that - fixing it.